backend/src/zac/core/camunda/utils.py

```diff
@@ -21,12 +22,20 @@
 }
 
 
-def _resolve_assignee(username: str) -> User:
-    try:
-        user = User.objects.get(username=username)
-    except User.DoesNotExist:
-        user = User.objects.create_user(username=username)
-    return user
+def _resolve_assignee(name: str) -> Union[Group, User]:
+    user_or_group, _name = name.split(":")[0], ":".join(name.split(":")[1:])
```

Comment:
Can be simplified a bit, no?

```py
>>> "foo:bar:baz".split(":", 1)
['foo', 'bar:baz']
```

```suggestion
    user_or_group, _name = name.split(":", 1)
```